schema-extended/Taxonomy.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
This file is probably poorly named since it doesn't doesn't include the complete hierarchy of the arXiv taxonomy along with descriptions.